### ISSUE TITLE
Frontmatter tags are not included in markdown pod import

### ISSUE THREAD

**Issue by chmac** [dendronhq/dendron#2198]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [x] Pod.  Data import from / export to Dendron

## Describe the bug

- Joplin's markdown+frontmatter export includes frontmatter tags
- The tags are in the same format as dendron (ie: tags property, array of strings, one per tag)
- These tags are not included in the imported notes

## To Reproduce

```
---
title: An example note
tags:
  - tag_one
  - tag_two
---
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean sit amet nibh.
```

1. Create an example markdown file as above in an otherwise empty directory
2. Command Palette > import pod > dendron.markdown
3. Edit your `config.import.yml` file in `pods/dendron.markdown`
  1. Set the `src` and `vaultName`
4. Repeat step 2 to run the import

## Expected behaviour

Look at the example note created, it has no tags in frontmatter

## Context

Dendron v0.77.1

**Comment by SeriousBug** [comment]
Thanks for reporting this, I think this is somewhat of a duplicate of #659, but with a smaller scope so I think it's worth leaving this issue open. We'll take up this feature whenever possible.

**Comment by chmac** [comment]
This is / will be fixed by #2195, which I hadn't seen when I opened the issue.

**Comment by Harshita-mindfire** [comment]
This bug should now be fixed with Dendron 0.80. For more information on how this is handled, and how it can be configured when there are conflicting attributes, reference the documentation [here](https://wiki.dendron.so/notes/f23a6290-2dec-45dc-b616-c218ee53db6b.html#importfrontmatter)
This issue will stay open for a few more days in case anyone continues to experience this problem.
